gensim/models/utils_any2vec_fast.pyx

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env cython
+# cython: boundscheck=False
+# cython: wraparound=False
+# cython: cdivision=True
+# coding: utf-8
+
+def ft_hash(unicode string):
+    cdef unsigned int h = 2166136261
+    for c in string:
+        h ^= ord(c)
+        h *= 16777619
+    return h
+
+
+cpdef compute_ngrams(word, unsigned int min_n, unsigned int max_n):
+    cdef unicode extended_word = f'<{word}>'
```

Comment:
f-strings supports only in `3.6` (and we maintain `2.7`, `3.5`, `3.6`), please use simple concatenation (or any alternative) here.

Reply:
This is in Cython, which since 0.24 automatically generates cross-compatible C-code for it. It works fine under 2.7.

Reply:
wow, really? I didn't know about it, thanks for the information!

Reply:
You can read about it here: http://cython.readthedocs.io/en/latest/src/tutorial/strings.html#string-literals